jupyter_server/services/contents/handlers.py

```diff
@@ -367,4 +369,5 @@ async def post(self, path=""):
     (r"/api/contents%s/trust" % path_regex, TrustNotebooksHandler),
     (r"/api/contents%s" % path_regex, ContentsHandler),
     (r"/api/notebooks/?(.*)", NotebooksRedirectHandler),
+    (r"/api/yjs/(.*)", YDocWebSocketHandler),
```

Comment:
Maybe we should change the `/api/yjs/...` endpoint, since it is not specific to Yjs.
What about `/api/ydoc/...`?